.github/workflows/cmake.yml

```diff
@@ -16,11 +16,19 @@ jobs:
 
     steps:
     - uses: actions/checkout@v3
-
-    - name: Install apt package
+    - name: Install apt packages
       run: |
         sudo apt-get update
-        sudo apt-get install -y doxygen
+        sudo apt-get install -y doxygen libncurses5
+  
+    - name: Install clang-format-10
+      run: |
+        cd ${{github.workspace}}
```

Comment:
I was thinking about using this https://github.com/ssciwr/clang-format-wheel, which should be easier to setup.

Reply:
Nice - I hadn't seen this. The default `apt install clang-format` installs the latest version (`14.0` I believe) and does not have a target for `clang-format-10`. Hence this solution